client/my-sites/stats/summary/index.jsx

```diff
@@ -212,6 +213,10 @@ class StatsSummary extends Component {
 					/>
 				);
 				break;
+			case 'annualstats':
+				title = translate( 'Annual Stats' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Annual Site Stats' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).